SPECS/kernel-hyperv/kernel-hyperv.spec

```diff
@@ -274,6 +274,9 @@ ln -sf linux-%{uname_r}.cfg /boot/mariner.cfg
 %{_libdir}/perf/include/bpf/*
 
 %changelog
+* Mon Mar 29 2021 Andrew Phelps <anphel@microsoft.com> - 5.10.21.1-3
```

Comment:
Shouldn't we change the `ID1=`readelf -n vmlinux | grep "Build ID"` lines in this spec the same way it was done for the "regular" kernel?